tests/test_api.py

```diff
@@ -1,7 +1,83 @@
+import pytest
+import re
+from unittest.mock import patch
+from unittest.mock import MagicMock
+
 import api
 from enums import SearchBehavior
 
 
+@pytest.mark.asyncio
+@patch("api._get_entities")
+async def test_search_posts_with_mentions(get_entities):
+    get_entities.return_value = 0, []
+    mongo = MagicMock()
```

Comment:
I think we can use `spec=quart_motor.Motor` here for some extra guardrails :) 
https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock

Reply:
Oh cool! I didn't know about that